uisdk/src/main/java/com/karhoo/uisdk/screen/booking/BookingActivity.kt

```diff
@@ -265,7 +261,18 @@ class BookingActivity : BaseActivity(), AddressBarMVP.Actions, BookingMapMVP.Act
         when {
             resultCode == Activity.RESULT_OK && requestCode == REQ_CODE_BOOKING_REQUEST_ACTIVITY -> {
                 if (data?.hasExtra(CheckoutActivity.BOOKING_CHECKOUT_PREBOOK_TRIP_INFO_KEY) == true) {
-                    showPrebookConfirmationDialog(data)
+                    journeyDetailsStateViewModel.process(AddressBarViewContract.AddressBarEvent.ResetJourneyDetailsEvent)
+                    val tripInfo = data.getParcelableExtra<TripInfo>(CheckoutActivity.BOOKING_CHECKOUT_PREBOOK_TRIP_INFO_KEY)
```

Comment:
`getParcelableExtra` is deprecated

Reply:
@dianakarh deprecated since Tiramisu. We're gonna keep it until we decide to split the two implementations based on API version (not needed IMHO)